Did you forgot to publish @pezzo/common package? Getting error "Cannot find module @pezzo/common"

### Report

Updated to from previous releases

"@pezzo/client": "^0.2.0",
"@pezzo/integrations": "^0.1.8"

and when run build, getting an error

```
node_modules/@pezzo/client/src/client/Pezzo.d.ts:2:47 - error TS2307: Cannot find module '@pezzo/common' or its corresponding type declarations.

2 import type { CreatePromptExecutionDto } from "@pezzo/common";
```


### Expected behavior

Expect not to get such a build issue

### Steps to reproduce the problem

1. pull the latest version of client / integration packages and try to use it :)



### Logs (if applicable)

```
node_modules/@pezzo/client/src/client/Pezzo.d.ts:2:47 - error TS2307: Cannot find module '@pezzo/common' or its corresponding type declarations.

2 import type { CreatePromptExecutionDto } from "@pezzo/common";
```


### Pezzo version

client version 0.2.0

### How do you use Pezzo?

Kubernetes Deployment

--- a/libs/client/package.json
+++ b/libs/client/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@pezzo/client",
   "description": "TypeScript API client for Pezzo",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "type": "commonjs",
   "author": "Ariel Weinberger",
   "keywords": [

--- a/libs/client/src/client/Pezzo.ts
+++ b/libs/client/src/client/Pezzo.ts
@@ -1,6 +1,6 @@
 import axios, { AxiosInstance } from "axios";
 import { IntegrationBaseSettings, PromptExecutionStatus } from "../types";
-import type { CreatePromptExecutionDto } from "@pezzo/common";
+import type { CreatePromptExecutionDto } from "./create-prompt-execution.dto";
 
 export interface PezzoClientOptions {
   serverUrl?: string;

--- a/libs/client/src/client/create-prompt-execution.dto.ts
+++ b/libs/client/src/client/create-prompt-execution.dto.ts
@@ -1,0 +1,64 @@
+import { PromptExecutionStatus } from "@prisma/client";
+import {
+  IsDateString,
+  IsEnum,
+  IsNumber,
+  IsObject,
+  IsOptional,
+  IsString,
+} from "class-validator";
+
+export class CreatePromptExecutionDto {
+  @IsDateString()
+  timestamp: string;
+
+  @IsString()
+  environmentId: string;
+
+  @IsString()
+  promptVersionSha: string;
+
+  @IsEnum(PromptExecutionStatus)
+  status: PromptExecutionStatus;
+
+  @IsObject()
+  settings: Record<string, unknown>;
+
+  @IsObject()
+  variables: Record<string, unknown>;
+
+  @IsString()
+  content: string;
+
+  @IsString()
+  interpolatedContent: string;
+
+  @IsOptional()
+  @IsString()
+  result?: string;
+
+  @IsOptional()
+  @IsString()
+  error?: string;
+
+  @IsNumber()
+  duration: number;
+
+  @IsNumber()
+  completionCost: number;
+
+  @IsNumber()
+  completionTokens: number;
+
+  @IsNumber()
+  promptCost: number;
+
+  @IsNumber()
+  promptTokens: number;
+
+  @IsNumber()
+  totalTokens: number;
+
+  @IsNumber()
+  totalCost: number;
+}

